app/javascript/components/enrollment/steps/Exposure.js

```diff
@@ -225,16 +226,23 @@ class Exposure extends React.Component {
       .then(function() {
         // No validation issues? Invoke callback (move to next step)
         self.setState({ errors: {} }, async () => {
-          if (self.state.current.patient.jurisdiction_id !== self.state.originalJurisdictionId) {
+          if (parseInt(self.state.current.patient.jurisdiction_id) !== self.state.originalJurisdictionId) {
+            // If we set it back to the last saved value no need to confirm.
+            if (self.state.current.patient.jurisdiction_id === self.state.selected_jurisdiction) {
+              callback();
+              return;
+            }
             const originalJurisdictionPath = self.props.jurisdiction_paths[self.state.originalJurisdictionId];
-            const message = `You are about to change the assigned jurisdiction from ${originalJurisdictionPath} to ${self.state.jurisdiction_path}. Are you sure you want to do this?`;
+            const message = `You are about to change the assigned jurisdiction from ${originalJurisdictionPath} to ${self.state.jurisdiction_path}. Only users in ${self.state.jurisdiction_path} and its parent jurisdictions will be able to access this record. \n\nAre you sure you want to do this? \n\nThis will NOT be saved until the record is saved.`;
```

Comment:
Where did this language update come from?

Reply:
The first part was from Alex directly. The second part was my wording to address Kelly's concern that if someone cancels out before finalizing the record they might not realize the change will not be saved.